content/en/blogs/kubesphere-certificates.md

```diff
@@ -0,0 +1,116 @@
+|Tally Lee|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-tallylee.png) |
+|Xiaofei Yang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-yangxiaofei.png) |
+|Jiong Duan|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-duanjiong.png) |
+|Wanjun Lei|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-leiwanjun.png) |
+|Michael Yuan|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-michaelyuan.png) |
+|Wenhu Wang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-wangwenhu.png) |
+|Ke Zhou|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-zhouke.png) |
+
+## KubeSphere Ambassador
+
+KubeSphere Ambassador is awarded to technical evangelists who are helping grow the KubeSphere community. The following members have shared KubeSphere practical cases and technical articles in the KubeSphere community many times to help more users to get to know the application scenarios and best practices of KubeSphere.
+
+| Name | Certificate |
+| ---- | ---- |
+|Guoyou Li|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-liguoyou.png) |
+|Pahud|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-pahud.png) |
+|Hengjun Wei|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-weihengjun.png) |
+|Lei Xue|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-xuelei.png) |
+|Yang Yang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-yangyang.png)|
+|Xingmin Lu|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-luxingmin.png) |
+|Haitao Pan|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-panhaitao.png) |
+|Will Zhang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-willzhang.png) |
+|Xinglin Xue|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-xuexinglin.png) |
+|Haili Zhang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-zhanghaili.png) |
+
+## KubeSphere Contributor
+
+KubeSphere Contributor is for members who have participated in KubeSphere’s open-source project in 2020 or published high-quality technical blogs on the KubeSphere Chinese forum, as well as members who help to submit and fix bugs.。
```

Comment:
->
KubeSphere Contributors are for members who have participated in KubeSphere’s open-source projects in 2020 or published high-quality technical blogs on the KubeSphere Chinese forum, as well as members who help to submit and fix bugs.